### PR TITLE
fix formatting of review txs value

### LINKED ISSUE #13603: Symbol wrapping
**Describe the bug**
Coin/Token symbol should not be wrapped in the middle of word

**Expected behavior**

Screenshot 2024-08-02 at 11 00 40

**Screenshots now**
This is an example from tx review modal but I think that it can occur everywhere, so it would be fine to fix it in some global component

Screenshot 2024-08-02 at 10 19 09
Screenshot 2024-08-02 at 10 18 26
Screenshot 2024-08-02 at 10 16 17
Screenshot 2024-08-02 at 9 55 06

### PR DESCRIPTION
**Pull request by ericvalcik**
## Description

Fix wrapping of Coin/Token symbol in the `TransactionReviewSummary.tsx`.

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/13603

## Screenshots:

before:
image

after changes:
image

### CODE REVIEW

**Comment by enjojoy** [comment]
Please squash the commits and use conventional commits style for the commit message

File: packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewSummary.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const Headline = styled.div`
     font-size: 16px;
     font-weight: 600;
     margin-top: 20px;
-    word-break: break-all;
+    word-break: normal;
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewSummary.tsx:72]: `break-word` would fix overflowing
Some tokens can have 18 decimals and amount can be also huge
Screenshot 2024-09-08 at 8 17 51

File: packages/suite/src/components/suite/FormattedCryptoAmount.tsx [hunk 2]
```diff
@@ -90,7 +90,7 @@ export const FormattedCryptoAmount = ({
         <Container className={className}>
             {!!signValue && <Sign value={signValue} />}
             <Value data-testid={dataTest}>{formattedValue}</Value>
-            {formattedSymbol && <BlurUrls text={'\u00A0' + formattedSymbol} />}
+            {formattedSymbol && <BlurUrls text={' ' + formattedSymbol} />}
```
Comment (tomasklim) [inline packages/suite/src/components/suite/FormattedCryptoAmount.tsx:93]: nit:
```suggestion
            {formattedSymbol && <>
              {' '}
              <BlurUrls text={formattedSymbol} />
            </>}
```
Something like that would look better

**Review by tomasklim — CHANGES_REQUESTED** [review]
Hello, thanks for contribution. I suggest you to create branches in your fork instead of commiting to develop

**Comment by ericvalcik** [comment]
Thanks for the comments @tomasklim. I implemented the comments and also moved the changes to a different branch, which made me create a new PR, so please let's continue the review process here -> https://github.com/trezor/trezor-suite/pull/14216

**Pull request closed without merging**